src/Datasource/EntityTrait.php

```diff
@@ -1356,7 +1356,7 @@ public function source($alias = null)
             get_called_class() . '::source() is deprecated. ' .
             'Use setSource()/getSource() instead.'
         );
-        if (is_null($alias)) {
+        if ($alias === null) {
```

Comment:
:+1: People should just add my Spryker sniffer rule to the CakePHP rules repo :)

Reply:
+ PHP CS Fixer has `is_null` rule. Hopefully, one of them can be added.